src/main/java/uk/gov/hmcts/reform/hmc/data/HearingPartyEntity.java

```diff
@@ -24,6 +25,7 @@
 @Table(name = "hearing_party")
 @Entity
 @Data
+@EqualsAndHashCode(exclude = "organisationDetailEntity")
```

Comment:
Had to add this to break hashCode cyclic dependency that caused `StackOverFlowError`

Reply:
Removed in [204b934](https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/204b934d37f2d1ffc62ac4080bd9e718ed5234dc)